communication-icons.js

```diff
@@ -1,9 +1,21 @@
+/**
+@license
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
 
-$_documentContainer.innerHTML = `<iron-iconset-svg name="communication" size="24">
+$_documentContainer.innerHTML =
```

Comment:
We should use the `html` function for consistency, here and everywhere, i.e:
```
import {html} from '@polymer/polymer/lib/utils/html-tag.js';
const template = html``;
document.body.appendChild(template.content);
```

Reply:
Was this the comment about using `html` that we were talking about leaving alone?

Reply:
yah but only if you think makes sense. if it looks like it's already being downloaded, then consistency maybe wins?

Reply:
iron-icon (imported by this file) [imports `html`](https://github.com/PolymerElements/iron-icon/blob/a805f91aa94a575ab094807f26b44a5267a80b87/iron-icon.js#L16), so I'll switch these.

Reply:
These are all updated now.